src/preferences/dialog/dlgprefbroadcast.cpp

```diff
@@ -126,9 +126,9 @@ DlgPrefBroadcast::DlgPrefBroadcast(QWidget *parent, UserSettingsPointer _config)
     comboBoxEncodingFormat->setCurrentIndex(tmp_index);
 
     // Encoding channels combobox
-    comboBoxEncodingChannels->addItem(tr("Stereo"), BROADCAST_CHANNELS_STEREO);
+    comboBoxEncodingChannels->addItem(tr("Automatic"), BROADCAST_CHANNELS_AUTOMATIC);
```

Comment:
I've decided to implement stereo and mono modes too. It's ready and tested. will commit it later.